src/groups/mqb/mqbblp/mqbblp_queue.h

```diff
@@ -178,7 +178,7 @@ class Queue BSLS_CPP11_FINAL : public mqbi::Queue {
                       int                       ackWindowSize,
                       RemoteQueue::StateSpPool* statePool);
 
-    void convertToLocal();
+    void convertToLocal() BSLS_KEYWORD_OVERRIDE;
     void convertToRemote();
```

Comment:
`convertToLocal` is the only method that needs to be promoted to base class to make CQH usable with `mqbi::Queue`